ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -17,19 +17,34 @@ class Tag {
 export default class FormWorkerCreateWorkerLedComponent extends Component {
   // =services
   @service features;
+  @service browserObject;
 
   // =attributes
   @tracked generatedWorkerAuthToken;
-  @tracked clusterId;
+  @tracked clusterId = this.clusterIdFromURL;
   @tracked ipAddress;
   @tracked configFilePath;
   @tracked initialUpstreams;
   @tracked workerTags = A([]);
   @tracked newWorkerKey;
   @tracked newWorkerValue;
 
-  // =properties
+  get clusterIdFromURL() {
+    const hostname = this.browserObject.window.location.hostname;
```

Comment:
Perhaps location and hostname should be part of the browser service.

Reply:
https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/alias